documentation/library_guide/parallel_api/parallel_range_algorithms.rst

```diff
@@ -0,0 +1,96 @@
+Parallel Range Algorithms
+#########################
+
+C++20 introduces the `Ranges library <https://en.cppreference.com/w/cpp/ranges>`_ and
+`range algorithms <https://en.cppreference.com/w/cpp/algorithm/ranges>`_ as a modern paradigm for expressing
+generic operations on data sequences.
+
+|onedpl_long| (|onedpl_short|) extends it with *parallel range algorithms* which can be used with the standard range
+classes to leverage |onedpl_short| ability of parallel execution on both the host computer and data parallel devices.
+
+oneDPL only supports random access ranges, because they allow simultaneous constant-time access to elements
+at any position in the range. This enables efficient workload distribution among multiple threads or processing units,
+which is essential for achieving high performance in parallel execution.
+
+.. Note::
+
+  The use of parallel range algorithms requires C++20 and the C++ standard libraries coming with GCC 10 (or higher),
+  Clang 16 (or higher) and Microsoft* Visual Studio* 2019 16.10 (or higher).
+
+Supported Range Views
+---------------------
+
+`Views <https://en.cppreference.com/w/cpp/ranges/view>`_ are lightweight ranges typically used to describe
+data transformation pipelines. The C++20 standard defines two categories of standard range views: factories
```

Comment:
Can we move factories and adaptors to bullet points? Example:
...standard range views:

- Factories
- Adaptors